docs/api/manual/UseCases.md

```diff
@@ -54,46 +49,42 @@ typedef struct {
 } UvisorPageTable;
 ```
 
-If enough free pages are available, the allocator will mark them as in use, zero them, and write each page origin into the page table.
+If enough free pages are available, the allocator will mark them as in use, zero them and write each page origin into the page table.
 
-The allocator returns an error code, if the page table is not formatted correctly.
-The requested `page_size` must be a equal to `UVISOR_PAGE_SIZE`, and all of the
-page table memory must be owned by the calling security context.
+The allocator returns an error code if the page table is not formatted correctly. The requested `page_size` must be equal to `UVISOR_PAGE_SIZE`, and the calling security context must own all of the page table memory.
 
-Note that the returned pages are **not** guaranteed to be allocated consecutively.
-It is the responsibility of the tier-2 allocator to make sure that memory requests for continous memory, that exceed the requested page size, are blocked.
+Note that there is **no** guarantee of the consecutive allocation of returned pages. It is the responsibility of the tier-2 allocator to make sure that memory requests for continuous memory that exceed the requested page size are blocked.
 
 <!--
->>> Comment: This concept has been rejected for now, since this is prone to fragmentation.
+>>> Comment: This concept has been rejected for now because this is prone to fragmentation.
```

Comment:
To what does this comment refer? Just the note above it? The whole section? If it doesn't work, should we take it out of the docs?